artifactory.py

```diff
@@ -2084,6 +2082,163 @@ def symlink_to(self, target, target_is_directory=False):
         raise NotImplementedError()
 
 
+class ArtifactoryBuild(ArtifactoryPath):
+    def __new__(cls, *args, **kwargs):
+        """
+        pathlib.Path overrides __new__ in order to create objects
+        of different classes based on platform. This magic prevents
+        us from adding an 'auth' argument to the constructor.
+        So we have to first construct ArtifactoryPath by Pathlib and
+        only then add auth information.
+        """
+        obj = super().__new__(cls, *args, **kwargs)
+        obj.project = kwargs.get("project", "")
+        return obj
+
+    @property
+    def builds(self):
+        """
+        Get all available builds on Artifactory
+        :return: (list) list of available build names
+        """
+        all_builds = []
+        url = ""
+        if self.project:
+            url = f"?project='{self.project}'"
+
+        obj = self.joinpath(url)
+        resp = self._accessor.get_response(obj).json()
+
+        if "builds" in self._get_build_api_response(url):
+            for build in resp["builds"]:
+                all_builds.append(build["uri"][1:])
+
+        return all_builds
+
+    def get_build_runs(self, build_name):
```

Comment:
Does `get_build_runs` return list of `build_number` in the build_name?

Reply:
It returns a dictionary with build names and dates